### PR TITLE
Image mirroring for kubevirt-machine-controllers 4.7

### PR DESCRIPTION
**Pull request by rgolangh**
Signed-off-by: Roy Golan <rgolan@redhat.com>

[OCPCNV-60](https://issues.redhat.com/browse/OCPCNV-60)

### CODE REVIEW

File: core-services/image-mirroring/openshift/mapping_origin_4_7 [hunk 1]
```diff
@@ -183,6 +183,7 @@ registry.svc.ci.openshift.org/ocp/4.7:ovirt-csi-driver quay.io/openshift/origin-
 registry.svc.ci.openshift.org/ocp/4.7:ovirt-csi-driver-operator quay.io/openshift/origin-ovirt-csi-driver-operator:4.7 quay.io/openshift/origin-ovirt-csi-driver-operator:4.7.0
 registry.svc.ci.openshift.org/ocp/4.7:ovirt-installer quay.io/openshift/origin-ovirt-installer:4.7 quay.io/openshift/origin-ovirt-installer:4.7.0
 registry.svc.ci.openshift.org/ocp/4.7:ovirt-machine-controllers quay.io/openshift/origin-ovirt-machine-controllers:4.7 quay.io/openshift/origin-ovirt-machine-controllers:4.7.0
+registry.svc.ci.openshift.org/ocp/4.7:kubevirt-machine-controllers quay.io/openshift/origin-kubevirt-machine-controllers:4.7 quay.io/openshift/origin-kubevirt-machine-controllers:4.7.0
```
Comment (smarterclayton) [inline core-services/image-mirroring/openshift/mapping_origin_4_7:186]: The file is supposed to be alphabetical

Comment (rgolangh) [reply core-services/image-mirroring/openshift/mapping_origin_4_7]: done

**Comment by rgolangh** [comment]
seems like kubevirt machine controllers is mapped already https://github.com/openshift/release/blob/0ac2cccd3e33a748523cb484a20ea27307a3f8b6/core-services/image-mirroring/openshift/mapping_origin_4_7#L140

/close

**Pull request closed without merging**